notebooks/week2/03.mlflow_experiment.py

```
@@ -0,0 +1,50 @@
+mlflow.set_experiment_tags({"repository_name": "child-health"})
+
+# COMMAND ----------
+experiments = mlflow.search_experiments(
+    filter_string="tags.repository_name='child-health'",
+)
+print(experiments)
+
+# COMMAND ----------
+with open("mlflow_experiment.json", "w") as json_file:
+    json.dump(experiments[0].__dict__, json_file, indent=4)
+    
```

Ruff

Blank line contains whitespace

Remove whitespace from blank line

(W293)